lib/DefinePlugin.js

```diff
@@ -99,24 +119,23 @@ class DefinePlugin {
 									 */
 									if (recurse) return;
 									recurse = true;
-									const res = parser.evaluate(code);
+									const res = parser.evaluate(toCode(code, parser));
 									recurse = false;
 									res.setRange(expr.range);
 									return res;
 								});
-							parser.hooks.expression
-								.for(key)
-								.tap(
-									"DefinePlugin",
-									/__webpack_require__/.test(code)
-										? ParserHelpers.toConstantDependencyWithWebpackRequire(
-												parser,
-												code
-											)
-										: ParserHelpers.toConstantDependency(parser, code)
-								);
+							parser.hooks.expression.for(key).tap("DefinePlugin", expr => {
+								code = toCode(code, parser);
```

Comment:
use a new local variable to assign to

Reply:
Doh. Sorry for wasting your time.
I will update now, and move on to the test suite. That may take a few days as I'm quite busy atm, but I'll get there.